pkg/odo/cli/project/delete.go

```diff
@@ -61,6 +61,8 @@ func (pdo *ProjectDeleteOptions) Validate() (err error) {
 
 // Run runs the project delete command
 func (pdo *ProjectDeleteOptions) Run() (err error) {
+
+	pdo.Context.Client.Namespace = pdo.projectName
```

Comment:
should this happen as part of context initialization?

Reply:
no, its not required during context initialization

Reply:
ack